addons/mail/static/src/core/common/message.js

```diff
@@ -310,6 +311,22 @@ export class Message extends Component {
         return this.props.thread.id === "inbox";
     }
 
+    get translatable() {
+        return (
+            this.store.hasMessageTranslationFeature &&
+            !this.env.inChatWindow &&
```

Comment:
I the goal is to restrict this feature to the chatter, it should be `this.env.inChatter`.
I don't see why can't have this feature globally? A livechat operator could find some good use of this.

Reply:
We briefly discussed your comment with Alex and we are happy to introduce this feature only in the chatter now. I will also make sure that PAJU is aware that we can widen the usage of translations with very little extra efforts in the futur.